client/landing/gutenboarding/onboarding-block/style-preview/font-select.tsx

```diff
@@ -3,40 +3,39 @@
  */
 import * as React from 'react';
 import { Button } from '@wordpress/components';
+import { ValuesType } from 'utility-types';
+import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
-import { ValuesType } from 'utility-types';
-
-export const fontPairings = [
-	[ 'Cabin', 'Raleway' ],
-	[ 'Chivo', 'OpenSans' ],
-	[ 'Playfair', 'FiraSans' ],
-	[ 'Arvo', 'Montserrat' ],
-	[ 'SpaceMono', 'Roboto' ],
-] as const;
+import { fontPairings } from '../../constants';
 
 interface Props {
 	onSelect: ( selection: ValuesType< typeof fontPairings > ) => void;
 	selected: ValuesType< typeof fontPairings >;
 }
 const FontSelect: React.FunctionComponent< Props > = ( { onSelect, selected } ) => (
 	<div className="style-preview__font-options">
-		<ul>
-			{ fontPairings.map( fonts => {
-				const [ a, b ] = fonts;
-				const isSelected = fonts === selected;
-				return (
-					<li key={ a + b }>
-						<Button onClick={ () => onSelect( fonts ) }>
-							<b>{ a }</b>&nbsp;/&nbsp;{ b }
-							{ isSelected && ' *' }
-						</Button>
-					</li>
-				);
-			} ) }
-		</ul>
+		{ fontPairings.map( fonts => {
+			const isSelected = fonts === selected;
+			const [
+				{ title: a, fontFamily: fontFamilyA },
+				{ title: b, fontFamily: fontFamilyB },
+			] = fonts;
+
+			return (
+				<Button
+					className={ classnames( 'style-preview__font-option', { 'is-selected': isSelected } ) }
+					onClick={ () => onSelect( fonts ) }
+					key={ a + b }
+				>
+					<span style={ { fontFamily: fontFamilyA, fontWeight: 700 } }>{ a }</span>
+					&nbsp;/&nbsp;
```

Comment:
Using `margin` instead of `&nbsp;` gives finer control over how much whitespace between `/`, sometimes fonts with different letter spacing makes the `/` looks off balance. But I think the more noticeable issue is that because different font pairings have different base height, I think design team @olaolusoga might want to think about using one line for font A and another line for font B instead.
![image](https://user-images.githubusercontent.com/1287077/76854585-52abfc80-688a-11ea-851d-3e2027588e56.png)

Reply:
https://github.com/Automattic/wp-calypso/issues/40206